src/psij/web/css/whole.css

```diff
@@ -0,0 +1,2 @@
12pt;padding-left:8pt;width:600pt}.ansi-bright-green-fg{color:#1e8c00}.ansi-bright-blue-fg{color:#0000b4}.ansi-bright-red-fg{color:red}.ansi-red-fg{color:#b40000}#status-calendar table{position:relative;width:100%;border-spacing:0}#status-calendar tr:first-child th{position:sticky;top:0pt}#status-calendar tr:first-child th{background-color:#DE6261;color:white;font-weight:bold}#status-calendar tr td,#status-calendar tr th{background-color:#e2e2e2;border-bottom:1px solid #cfcfcf}#status-calendar tr:nth-child(even) td,#status-calendar tr:nth-child(even) th{background-color:#f2f2f2}.calendar-month{height:14pt;font-weight:normal;font-size:9pt;padding-top:5px;width:60pt}.calendar-day{height:18pt;font-weight:normal;font-size:11pt;padding-bottom:4pt}.calendar-site{text-align:left;font-weight:normal;font-size:11pt;height:24pt;padding:4pt}.calendar-tile{margin-left:auto;margin-right:auto;display:flex;flex-direction:column;flex-wrap:wrap;height:10pt;align-content:center}.calendar-bubble{border-radius:100pt;border:0.5px solid rgba(0,0,0,0.2);display:inline-block}.calendar-bubble.state-good{background-color:#6aff4d}.calendar-bubble.state-somewhat-bad{background-color:#f2db49}.calendar-bubble.state-really-bad{background-color:#ffb0a0}.calendar-bubble.bubble-large{width:8pt;height:8pt;margin-right:2pt}.calendar-bubble.bubble-small{width:3pt;height:3pt;margin-right:1pt;margin-bottom:1pt}.calendar-tile.state-somewhat-bad{background-color:#f0f0d0}.calendar-tile.state-really-bad{background-color:#ff4d4d}.calendar-tile.state-good{background-color:#6aff4d}.calendar-tile.state-unknown{background-color:#e0e0e0}.bg{background-color:#f0f0f0}.calendar-cell{text-align:center}.IntroBlurb{background-color:#f1f0f0;border:1px solid #e2e2e2;border-radius:5px;padding:9px;margin:10px 0 10px 0;font-size:13px;color:#000000;position:relative}.IntroBlurb .main_slot{text-align:left}.IntroBlurb .CloseMe{position:absolute;top:7px;right:10px;color:#999999;font-size:17px;cursor:pointer}.open_close_arrow{border-left:8px solid transparent;border-right:8px solid transparent;border-top:8px solid #444444;margin-left:8px;cursor:pointer}.open_close_arrow[position="otherPos"]{display:none}.open_close_arrow[position="oneRowOnly"]{display:none}.open_close_arrow.open{position:relative;left:5px;border-top:8px solid transparent;border-bottom:8px solid transparent;border-left:8px solid #444444}#settings{position:relative;top:-8px;right:4px;border:1px solid #bE4241;border-bottom-left-radius:5px;border-bottom-right-radius:5px;box-shadow:2px 2px 2px;-moz-box-shadow:2px 2px 2px;-webkit-box-shadow:2px 2px 2px;cursor:pointer}#settings:active{box-shadow:0 0 0;-moz-box-shadow:0 0 0;-webkit-box-shadow:0 0 0;position:relative;top:-6px;right:2px;cursor:pointer}.main-exaworks-logo{position:absolute;top:0;left:12px;width:120px;height:27px;z-index:200;padding:5px 15px;background:rgba(255,255,255,0.15);border:solid 1px;border-color:#fff;border-color:rgba(255,255,255,0.75);border-top:0;border-radius:0 0 0.35em 0.35em}
```

Comment:
I agree with the simplicity principle.  But, SCSS asks so little of us and gives back so much more.  Whereas something like Vue or other "modern" javascript frameworks are incredibly complex and give back very little advantage.  If anything, I think we should use less Vue.

Reply:
I think I explained rather clearly and with specifics why SCSS is a problem. Adding npm to the list of dependencies is not "little". And you are not explaining how SCSS is "much more". Does it result in a significant simplification of our css files?

Adding Vue to the conversation or challenging the usefulness of widely used libraries or of all modern JS frameworks is not helpful.

Reply:
The reason I brought up Vue was because I see Vue as being far more complex than SCSS.  SCSS is very light weight and helps simplify things.


Here are a few of the benefits of SCSS:
* You can nest selectors and so you can specify all the CSS selectors underneath another one and easily see which rules are under which divs
* You can specify site wide color variables 
* We get aggregation and minification (something we want anyway) by default so our site loads faster (fewer http requests and smaller files).  and don't worry the browser is able to unpack it so it's still just as easy to debug

Reply:
Vue does not require me to install anything, whereas SCSS needs a compiler. You can probably use sass.js for that, which would be an improvement. However, it is still a technological change that should have some **specific** benefits **for this project**.

I am not looking for the "benefits of SCSS". I am looking, specifically, to see how adding this technology helps us. And what you list are not specific issues to the dashboard but general talking points for SCSS. Because you can also specify global color variables in plain CSS. As for loading time, all resources are loaded in parallel. Style.css takes about 66ms to load, whereas jquery takes 270 ms. You will not see one yota of benefit in making the CSS smaller unless it takes more than what the longest resource takes (so at least more than 270 ms).